library/Mockery/Expectation.php

```diff
@@ -331,7 +340,7 @@ protected function _matchArg($expected, &$actual)
         if ($expected instanceof \Mockery\Matcher\MatcherAbstract) {
             return $expected->match($actual);
         }
-        if (is_a($expected, '\Hamcrest\Matcher') || is_a($expected, '\Hamcrest_Matcher')) {
+        if ($expected instanceof \Hamcrest\Matcher || $expected instanceof \Hamcrest_Matcher) {
```

Comment:
@aik099 :+1:  you're right, it makes code reviews more difficult. I'll consider it next time :)